### ISSUE TITLE
Use own data to train this model

### ISSUE THREAD

**Issue by Yujia-Tang** [bailvwangzi/repulsion_loss_ssd#16]
I try to train my own data using this model, but there is easy to be NaN loss. I have modified my learning rate enough small and double the batch size. It does not work. And I want to transfer the loss to Yolo model. Is it feasible?

**Comment by wyz0916-gitHub** [comment]
Hi,I meet the same problem.Have you finished?
